/partol_node:
  ros__parameters:
    initial_point: [0.0,0.0,0.0]
    target_point: [
      0.0,0.0,0.0,
      2.0,-0.3,0.0,
      1.0,1.0,1.57,
      0.6,-1.5,0.0,
      0.0,0.0,1.57
    ]
    use_sim_time: false